routes/verifyPhoneRoutes.js

```diff
@@ -0,0 +1,16 @@
+const router = require("express").Router();
+
+const verifyPhone = require("../services/verifyPhone");
+
```

Comment:
I like how you're using the `Verify Phone` feature! this can stop possible spam hits against our online balance for texting.